static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1,69 +1,164 @@
-$quote-tail: 25px; // ********** Article Type **********
+$quote-tail: 25px;
+@mixin multiline($number) {
+    content: '';
+    display: block;
+    width: 100%;
+    margin-bottom: $gs-baseline/4;
+    background-image: repeating-linear-gradient(#dcdcdc, #dcdcdc 1px, #ffffff 1px, #ffffff 4px);
+    height: $number * 4px;
+}
+// hack to hide left/right page margins, temp
```

Comment:
could we just remove them? is there a plan to?

Reply:
yes but how can we without removing them from live?